Frontend/CO.CDP.OrganisationApp/Pages/Registration/OrganisationIdentification.cshtml.cs

```diff
@@ -137,4 +171,15 @@ public IActionResult OnPost()
         return RedirectToPage("./OrganisationRegisteredAddress");
     }
 
+    private RegistrationDetails VerifySession()
+    {
+        var registrationDetails = session.Get<RegistrationDetails>(Session.RegistrationDetailsKey);
+        if (registrationDetails == null)
+        {
+            //show error page (Once we finalise)
```

Comment:
Or... redirect to the page that starts gathering information.

Reply:
Its a good idea, I will discuss this with Devs as we havent added this logic on any of the pages yet.